csharp/autobuilder/Semmle.Autobuild.Shared/MsBuildRule.cs

```diff
@@ -15,12 +15,12 @@ internal static class MsBuildCommandExtensions
         /// <returns></returns>
         public static CommandBuilder MsBuildCommand(this CommandBuilder cmdBuilder, IAutobuilder<AutobuildOptionsShared> builder)
         {
-            var isArmMac = builder.Actions.IsMacOs() && builder.Actions.IsArm();
+            var IsRunningOnAppleSiliconMac = builder.Actions.IsMacOs() && builder.Actions.IsRunningOnAppleSilicon();
```

Comment:
```suggestion
            var IsRunningOnAppleSiliconMac = builder.Actions.IsRunningOnAppleSilicon();
```

Reply:
maybe: even inline this into the ?: operator

Reply:
Thanks.